LgFaceRecDemoTCP_Jetson_NanoV2/src/key_manager.cpp

```diff
@@ -0,0 +1,57 @@
+#include <sys/types.h>
+extern "C" {
+#include <keyutils.h>
+}
+#include <time.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <errno.h>
+#include "key_manager.h"
+
+int cctv_request_key(const char *desc, unsigned char *key, int *len)
+{
+    char auth_key_payload[256] = { 0, };
```

Comment:
```suggestion
    constexpr unsigned int KEY_SIZE = 256;
    char auth_key_payload[KEY_SIZE] = { 0, };
```
매직넘버가 쓰이는 것이 신경쓰이는데, 위와 같이 수정하면 어떨까요? (아래 쪽 사용 코드쪽 suggestion 포함)